pkg/apiclient/error-translating-cluster-workflow-template-service-client.go

```diff
@@ -0,0 +1,65 @@
+package apiclient
```

Comment:
Is one more wrapper need for just translating errors?

Reply:
correct